Dev

SurajThotakura opened the pull request:
Implemented the test mechanism with splitting JS.
Implemented a caret with smooth transition.
Added colors for error.
By changing the method to splitting the UX issues are resolved.

Known issue, test restart not working.

SurajThotakura committed:
Splitting method

Updated the test mechanism with splitting js.
Added a caret and colours for error.

This update solves the UX issues.

Known issue, restarting test not working as expected.

SurajThotakura committed:
Delete index.html

SurajThotakura committed:
Delete index.js

SurajThotakura committed:
Revert "Update index.html"

This reverts commit 095e11ad37e216ae0739e7550a052c1a2c106902.

SurajThotakura committed:
Splitting method

Updated the test mechanism with splitting js.
Added a caret and colours for error.

This update solves the UX issues.

Known issue, restarting test not working as expected.

SurajThotakura committed:
Merge branch 'dev' of https://github.com/SurajThotakura/boardsmash-js into dev

SurajThotakura committed:
Significant Update Splitting method

SurajThotakura committed:
test restart fix

akhil0001 reviewed (approved):
If this is working fine, we can merge this. This PR has been hanging around for quite a bit of time

SurajThotakura committed:
fixed bugs

1. Fixed restart bug
2. Timer starts after the first keydown event
3. Caret position fix.

SurajThotakura committed:
Merge branch 'main' into dev

SurajThotakura committed:
conflict resolve

SurajThotakura committed:
conflict update
